schemas/src/test/resources/test-profiles/valid/simple-data-constraints-nested-7level.json

```diff
@@ -1,4 +1,4 @@
-﻿{
+{
```

Comment:
Is this change removing the BOM? Does the validator work when the BOM is included in the JSON?

Reply:
Bill Of Materials ?
dont think we need one.